docs/apache-airflow/templates-ref.rst

```diff
@@ -47,16 +47,16 @@ Variable                                    Type                  Description
                                                                   | Example: ``20180101T000000+0000``.
 ``{{ ts_nodash }}``                         str                   | Same as ``{{ dag_run.logical_date | ts_nodash }}``.
                                                                   | Example: ``20180101T000000``.
-``{{ prev_data_interval_start_success }}``  `pendulum.DateTime`_  | Start of the data interval of the prior successful DAG run.
+``{{ prev_data_interval_start_success }}``  `pendulum.DateTime`_  | Start of the data interval of the prior successful `DAG Run`_.
                                             | ``None``            | Added in version 2.2.
-``{{ prev_data_interval_end_success }}``    `pendulum.DateTime`_  | End of the data interval of the prior successful DAG run.
+``{{ prev_data_interval_end_success }}``    `pendulum.DateTime`_  | End of the data interval of the prior successful `DAG Run`_.
                                             | ``None``            | Added in version 2.2.
-``{{ prev_start_date_success }}``           `pendulum.DateTime`_  Start date from prior successful DAG run (if available).
+``{{ prev_start_date_success }}``           `pendulum.DateTime`_  Start date from prior successful `DAG Run`_ (if available).
                                             | ``None``
-``{{ dag }}``                               DAG                   The currently running DAG.
-``{{ task }}``                              BaseOperator          | The currently running task.
+``{{ dag }}``                               DAG                   The currently running `DAG`_. You can read more about DAGs in :doc:`DAGs <core-concepts/dags>`.
```

Comment:
```suggestion
``{{ dag }}``                               DAG                   The currently running :class:`~airflow.models.dag.DAG`. You can read more about DAGs in :doc:`DAGs <core-concepts/dags>`.
```

I think you can just do this without the indirection below. It’s just preference though.